pypeit/core/coadd.py

```diff
@@ -567,6 +564,31 @@ def solve_poly_ratio(wave, flux, ivar, flux_ref, ivar_ref, norder, mask = None,
     flux_med, ivar_med = median_filt_spec(flux, ivar, mask, med_width)
     flux_ref_med, ivar_ref_med = median_filt_spec(flux_ref, ivar_ref, mask_ref, med_width)
 
+    if 'poly' in model:
+        #guess = np.append(ratio, np.zeros(norder))
+        yval = flux_ref_med
+        yval_ivar = ivar_ref_med
+        scale_mask = np.ones_like(flux_ref_med, dtype=bool)
+    elif 'square' in model:
+        #guess = np.append(np.sqrt(ratio), np.zeros(norder))
+        yval = np.sqrt(flux_ref_med + (flux_ref_med < 0))
+        yval_ivar = 4.0*flux_ref_med
```

Comment:
I think this should be
`yval_ivar = 4.0*flux_ref_med * ivar_ref_med`

Reply:
Good catch. Fixed.